src/applications/mhv-medical-records/tests/e2e/accelerated/pages/Allergies.js

```diff
@@ -20,12 +21,13 @@ class Allergies {
       }
       req.reply(allergiesData);
     }).as('allergies-list');
+    MedicalRecordsLandingPage.uumIntercept();
```

Comment:
Note: I add the intercept for UUM to get rid of any errors, but we don't test it in the E2E tests.